src/components/media_manager/src/socket_streamer_adapter.cc

```diff
@@ -132,7 +109,7 @@ bool SocketStreamerAdapter::SocketStreamer::Send(
   LOG4CXX_AUTO_TRACE(logger);
   size_t ret;
   if (is_first_frame_) {
-    ret = send_socket_.Send(header_.c_str(), header_.size(), MSG_NOSIGNAL);
+    ret = client_socket_.Send(header_.c_str(), header_.size());
```

Comment:
Could we update a ``clinet_socket`` interface with ``Send(const std::vector<>&)``? 
I think this override usage will make code more readable.